### PR TITLE
Add Foundry To Chains Export

### PR DESCRIPTION
**Pull request by carllippert**
Foundry is becoming more popular. This makes it easy for all.

## Description

Make Foundry as easy as hardhat

## Additional Information

- [x] I read the [contributing docs](/tmm/wagmi/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

Your ENS/address:

carllippert.eth

### CODE REVIEW

File: packages/core/src/constants/chains.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const chainId = {
   arbitrumRinkeby: 421_611,
   localhost: 1_337,
   hardhat: 31_337,
+  foundry: 31337
```
Comment (jxom) [inline packages/core/src/constants/chains.ts:34]:
```suggestion
  foundry: 31_337
```

**Review by tmm — CHANGES_REQUESTED** [review]
We should export this from the tree shakable chains entrypoint in each package.

Also, should we call this Foundry Anvil?

**Comment by carllippert** [comment]
Was thinking the same about Anvil naming! Someone from that team probably has an opinion but don't know them.

**Comment by tmm** [comment]
Closing in favor of https://github.com/tmm/wagmi/pull/570

**Pull request closed without merging**